DEVELOPING.md

```diff
@@ -136,6 +136,7 @@ shuttle-persist = { path = "[base]/shuttle/resources/persist" }
 shuttle-shared-db = { path = "[base]/shuttle/resources/shared-db" }
 shuttle-secrets = { path = "[base]/shuttle/resources/secrets" }
 shuttle-static-folder = { path = "[base]/shuttle/resources/static-folder" }
+shuttle-service-info = { path = "[base]/shuttle/resources/service-info" }
```

Comment:
I think there might be 2 more places where this pattern needs adding (bit of a shame)

Reply:
3e3da5d186d3473281045f2d9a9211e20f374794

Reply:
Now it's just the CI file left I think

Reply:
Oops
b22fd60b0064db83e6f1a00e1016b1c0a169c645